dialect.go

```diff
@@ -298,7 +298,9 @@ func (m ClickHouseDialect) createVersionTableSQL() string {
       is_applied UInt8,
       date Date default now(),
       tstamp DateTime default now()
-    ) Engine = MergeTree(date, (date), 8192)`, TableName())
+    )
+	ENGINE = MergeTree()
+	  ORDER BY (version_id)`, TableName())
```

Comment:
Why change the sorting key from `date` to `version_id` ?

The sorting keys are either autoincrementing id's or default date at the time of insert.

Reply:
Fixed.